tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -384,3 +385,38 @@
                 MyManager = BaseManager.from_queryset(MyQuerySet)
                 class MyModel(models.Model):
                     objects = MyManager()
+
+
+-   case: from_queryset_custom_auth_user_model
+    disable_cache: true
```

Comment:
Let's add a comment why do we need this.
And a link to all the discussions somewhere above this test case.